### PR TITLE
:bug: Add error pop up and redirection for non existing workflow IDs

### PR DESCRIPTION
**Pull request by Bearbobs**
https://linear.app/n8n/issue/N8N-2621/show-error-when-workflow-of-execution-is-deleted

### CODE REVIEW

File: packages/editor-ui/src/views/NodeView.vue [hunk 1]
```diff
@@ -1633,11 +1633,21 @@ export default mixins(
...
+							// send error pop-up
+							this.$showMessage({
+								title: 'Error',
+								message: 'Could not find workflow',
+								type: 'error',
+							});
+						} else {
+							this.$titleSet(workflow.name, 'IDLE');
+							// Open existing workflow
+							await this.openWorkflow(workflowId);
```
Comment (mutdmour) [inline packages/editor-ui/src/views/NodeView.vue:1649]: just remove the comments, they don't add any valuable information the code does not already say

**Review by mutdmour — COMMENTED** [review]
lgtm. just a minor comment

File: packages/editor-ui/src/views/NodeView.vue (hunk 1 above)
Comment (Bearbobs) [reply packages/editor-ui/src/views/NodeView.vue]: done

**Review by mutdmour — APPROVED** [review]

**Comment by janober** [comment]
Thanks a lot. Got merged.

**Comment by janober** [comment]
Got released with `n8n@0.152.0`

**Pull request merged**